From: Jason Abele <jason.abele@gmail.com>
Date: Thu, 12 Nov 2015 18:21:10 -0800
Subject: move bt_rtk_hciattach back to bluetooth.target

---
 bt_rtk_hciattach.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bt_rtk_hciattach.service b/bt_rtk_hciattach.service
index a9f92c2..e4c2f6d 100644
--- a/bt_rtk_hciattach.service
+++ b/bt_rtk_hciattach.service
@@ -12,4 +12,4 @@ ExecStartPre=/sbin/bt_reset
 ExecStart=/sbin/rtk_hciattach -n -s 115200 /dev/ttyS1 rtk_h5
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=bluetooth.target
